styles/styles.css

```diff
@@ -84,9 +84,11 @@
   --boarder-trendign-news-card: #EAEAEA;
   --card-post-color: #6B6B6B;
   --card-post-boarder: #d7d7d7;
-  --modal-background-color: #000000BF;
+  --modal-background-color: #000000BF
```

Comment:
we can avoid block speicific global variables, since it is a color we can name it in a generic way